index.html

```diff
@@ -8,8 +8,8 @@
     </head>
     <body>
         <header>
-            <div>    
-                <a id="logo" href="/index.html"><img src="img/logo.png"/></a>
+            <div>
+                <a id="logo" href="index.html"><img src="img/logo.png"/></a>
                 <nav>
                     <a href="https://mic.am/">HOME<span>Back to home</span></a>
```

Comment:
It will be better to add "Home" text to another span, and giv it class like linkHeader, and give linkDescription class to "Back to home" text, and style them with nav a linkHeader and nav a linkDescription selectors.